packages/legacy/src/queries/providers.ts

```diff
@@ -261,6 +261,7 @@ export const usePatchProviderMutation = (
       queryClient.invalidateQueries('cluster-providers');
       queryClient.invalidateQueries('inventory-providers');
       queryClient.invalidateQueries('secrets');
+      queryClient.invalidateQueries('secret');
```

Comment:
not sure we have a query named `secret` , are you sure ?

Reply:
This added line is exactly what fixed the edit provider secret fields sync. 

While editing an existing provider, the `useSecretQuery` is called with the 'secret' query key: https://github.com/kubev2v/forklift-console-plugin/blob/e68ee2f064397ee254da107bde1463d9e45d754e/packages/legacy/src/queries/secrets.ts#L18 instead of calling  the `useSecretsQuery` with 'secrets' query key like in all other use cases https://github.com/kubev2v/forklift-console-plugin/blob/e68ee2f064397ee254da107bde1463d9e45d754e/packages/legacy/src/queries/secrets.ts#L45.

Reply:
NICE !